frontend/src/pages/projects/screens/projects/ProjectTableRow.tsx

```diff
@@ -47,10 +48,28 @@ const ProjectTableRow: React.FC<ProjectTableRowProps> = ({
             >
               {index === 0 ? (
                 <Td dataLabel="Name" rowSpan={notebookStates.length || 1}>
-                  <ResourceNameTooltip resource={project}>
-                    <ProjectLink project={project} />
-                  </ResourceNameTooltip>
-                  {owner && <Text component={TextVariants.small}>{owner}</Text>}
+                  <TableRowTitleDescription
+                    title={
+                      <ResourceNameTooltip resource={project}>
+                        <ProjectLink
+                          project={project}
+                          style={{
+                            fontSize: 'var(--pf-v5-global--FontSize--md)',
+                            fontWeight: 'var(--pf-v5-global--FontWeight--normal)',
+                          }}
+                        />
+                      </ResourceNameTooltip>
+                    }
```

Comment:
This is purposely going against the styling we have built into `TableRowTitleDescription` which is consistent for links across the app.
Why isn't this using the same style. ie same font size but bold
@tobiastal